### PR TITLE
[REM] l10n_in: remove audit trail feature

### PR DESCRIPTION
**Pull request by kasp-odoo**
Before this commit
==================
As the `account_audit_trail` module was separated from the `l10n_in` module, but there were duplicate views and models of the audit trail feature in the `l10n_in` module.

After this commit
=================
Removed duplicate views and models of the audit trail feature from the `l10n_in` module.

Related upgrade PR: odoo/upgrade#5462

task_id: 3326258

### CODE REVIEW

File: addons/l10n_in/models/company.py
Comment (rugo-odoo) [inline addons/l10n_in/models/company.py:1]: If I'm not mistaken, you are removing this behavior (i.e. not being allowed to deactivate the audit trail) completely. Is it intended ?

**Review by rugo-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_in/models/company.py
Comment (kasp-odoo) [reply addons/l10n_in/models/company.py:1]: Hello @rugo-odoo, this behaviour has been added to the `account_audit_trail` module at the setting level. So, it isn't required here.

Comment (rugo-odoo) [reply addons/l10n_in/models/company.py:1]: I think there is a difference tho. Here you can't deactivate it once it's active and the behavior in `account_audit_trail` you can't deactivate it once there are moves. So it is not really the same (if I'm not mistaken).

I don't have a problem with the change in behavior as long as it is intended.

Comment (kasp-odoo) [reply addons/l10n_in/models/company.py:1]: But now, it is not required at all in `l10n_in`.

**Comment by rugo-odoo** [comment]
LGTM (cc @william-andre)

**Comment by william-andre** [comment]
@robodoo r+

**Review by william-andre — APPROVED** [review]

**Pull request closed without merging**